src/pages/library/DataBrowser.js

```diff
@@ -1,11 +1,35 @@
+import _ from 'lodash/fp'
+import { Fragment, useState } from 'react'
 import { h } from 'react-hyperscript-helpers'
 import FooterWrapper from 'src/components/FooterWrapper'
+import { DelayedSearchInput } from 'src/components/input'
 import { libraryTopMatter } from 'src/components/library-common'
 
 
 const DataBrowser = () => {
+  const items = ['test1', 'test2']
+
+  const [searchFilter, setSearchFilter] = useState()
+
+  const filterBySearch = items => {
+    const lowerSearch = _.toLower(searchFilter)
+    return _.isEmpty(lowerSearch) ?
+      items :
+      _.filter(item => _.includes(lowerSearch, item), items)
```

Comment:
```suggestion
    return _.isEmpty(lowerSearch) ? items : _.filter(item => _.includes(lowerSearch, item), items)
```
Optional. We value concision, so separating out these lines might not be worth doing. This is a very straightforward concept, so it should probably take up as little space as possible.

Reply:
Actually, Utils.textMatch might be more appropriate here. Please take a look.